kernel/linux/linux.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+import stdlib
+import stdlib.build
+import stdlib.split.system
+from stdlib.template import autotools
+from stdlib.template.make import make
+from stdlib.manifest import manifest
+
+
+def install_linux():
+    build = stdlib.build.current_build()
+
+    boot_dir = os.path.join(build.install_cache, 'boot')
+    modules_dir = os.path.join(build.install_cache, 'usr')
+    include_dir = os.path.join(build.install_cache, 'usr')
+
+    os.makedirs(boot_dir, exist_ok=True)
+    os.makedirs(modules_dir, exist_ok=True)
+    os.makedirs(include_dir, exist_ok=True)
+
+    # make(f'INSTALL_PATH={boot_dir}', 'install')
```

Comment:
Should these commented lines be removed ?

Reply:
I left them on purpose.
This manifests only compiles `kernel/linux-dev` for now, not `kernel/linux`, and those lines are the one that we will uncomment to enable the later one.